ice_setup/ice.py

```diff
@@ -567,6 +575,7 @@ def create_repo_file(cls, template_name, repo_url, gpg_url, file_name=None, **kw
             contents = template.format(
                 gpg_url=gpg_url,
                 repo_url=repo_url,
+                gpg_check=1 if use_gpg else 0,
```

Comment:
After reading around a bit, I'm definitely on the side of changing the signature.

The best times I see for adding things in **kw are:

1. passing arbitrary kwargs to another function (as in a wrapper or decorator)
2. actually needing iterate over a bunch of kwargs that you don't know what they are going to be (like adding them to a list or dict)
3. subclassing/extending methods and protecting yourself against future changes.

The subprocess wrappers in this file are a great example of 1 and 3.

I found this SO discussion helpful: http://stackoverflow.com/questions/1415812/why-use-kwargs-in-python-what-are-some-real-world-advantages-over-using-named

@alfredodeza how strong do you feel about this?

Also, I can answer my own question about "what about the others"?  I didn't actually look at those when I wrote that -- the method you commented on is the only that has an existing `**kw` in the signature.